sharepoint-adfs/dsc/ConfigureFEVM.ps1

```diff
@@ -243,9 +243,14 @@ configuration ConfigureFEVM
         #**********************************************************
         # Install applications using Chocolatey
         #**********************************************************
-        cChocoInstaller InstallChoco
-        {
-            InstallDir = "C:\Program Files\Choco"
+        # cChocoInstaller InstallChoco
+        # {
+        #     InstallDir = "C:\Program Files\Choco"
+        # }
+        # Workaround for issue https://github.com/chocolatey/cChoco/issues/151
+        cChocoinstaller InstallChoco {
+            InstallDir            = "C:\Choco"
+            ChocoInstallScriptUrl = "https://gist.githubusercontent.com/artisticcheese/d934c1fb704a3e67b3c68283bcabca66/raw/9345bcb115ee7350172fa00085514212245a1c65/install.ps1"
```

Comment:
we can't pull from a private user account... you can host it with the sample, or I can put it in a static location in the subscription used for CI (which may be simpler but require manual updates (which may be fine if this is temporary))

Reply:
Understood, let me copy the script to the DSC folder and push a new commit

Reply:
Done. I expect that the CI builds will fail since DSC won't be able to download "https://github.com/Azure/azure-quickstart-templates/blob/master/sharepoint-adfs/dsc/InstallChocolatey.ps1" yet